### PR TITLE
Bulatkh/fix model init

### PR DESCRIPTION
**Pull request by annanda**

### CODE REVIEW

**Comment by annanda** [comment]
Fix issue with not relevant parameters in state_dict (such as loss.weight), fix Mock XRoom Writer with new data format.
